### PR TITLE
fix helm and surveys  importing

### PR DESCRIPTION
**Pull request by dsever**
To have working helm I added minor fixes (in commits), also quotas needed to be added to properly import surveys

**Note: DefectDojo is now on Python3 and Django 2.2.1 Please submit your pull requests to the 'dev' branch as the 'legacy-python2.7' branch is only for bug fixes. Any new features submitted to the legacy branch will be ignored and closed.**

When submitting a pull request, please make sure you have completed the following checklist:

- [ ] Your code is flake8 compliant 
- [ ] Your code is python 3.5 compliant
- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
- [ ] Model changes must include the necessary migrations in the dojo/dd_migrations folder.
- [ ] Add applicable tests to the unit tests.

### CODE REVIEW

**Comment by ptrovatelli** [comment]
I don't understand the changes on helm. See also this comment: https://github.com/DefectDojo/django-DefectDojo/pull/1752#issuecomment-589711248

To me this outside secret is mandatory or else you loose the secret when uninstalling the chart: while the database is still existing, it cannot be accessed.

File: helm/defectdojo/templates/django-ingress.yaml [hunk 1]
```diff
@@ -30,5 +30,5 @@ spec:
         - path: /
           backend:
             serviceName: {{ $fullName }}-django
-            servicePort: http
```
Comment (ptrovatelli) [inline helm/defectdojo/templates/django-ingress.yaml]: http worked for me; it's some kind of alias I think; is this change really necessary?

File: helm/defectdojo/templates/configmap.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ data:
   DD_ADMIN_MAIL: {{ .Values.admin.Mail | default "admin@defectdojo.local" }}
   DD_ADMIN_FIRST_NAME: {{ .Values.admin.FirstName | default "Admin" }}
   DD_ADMIN_LAST_NAME: {{ .Values.admin.LastName | default "User" }}
-  DD_ALLOWED_HOSTS: {{ .Values.host }}
+  DD_ALLOWED_HOSTS: "*"
```
Comment (ptrovatelli) [inline helm/defectdojo/templates/configmap.yaml]: what's wrong with secured by default? :)
it worked for me with Values.host, are you sure this change is necessary?

File: helm/defectdojo/templates/secret-postgresql.yaml [hunk 3]
```diff
@@ -12,6 +11,4 @@ metadata:
     helm.sh/resource-policy: keep
 type: Opaque
 data:
-  postgresql-password : {{ randAlphaNum 10 | b64enc | quote }}
-
-{{- end }}
+  postgresql-password : {{ .Values.postgresqlPassword | default "postgres" | b64enc | quote }}
```
Comment (ptrovatelli) [inline helm/defectdojo/templates/secret-postgresql.yaml]: this is not in values.yaml

File: helm/defectdojo/values.yaml [hunk 4]
```diff
@@ -66,7 +70,7 @@ django:
         cpu: 100m
         memory: 128Mi
   nodeSelector: {}
-  replicas: 1
+  replicas: 4
```
Comment (ptrovatelli) [inline helm/defectdojo/values.yaml]: that's a lot of replicas by default; this will be heavy for people deploying on minikube. why not overriding this parameter if you wish to change the value? the kubernetes.md already gives hint for production deployment with replicas.

**Comment by ptrovatelli** [comment]
@dsever can you post on slack or in a new issue detailed information about the problem you're experiencing with helm that made you think that those changes were necessary? with all the commands that you ran and the verifications that you made. 

It's possible that the doc has some holes but I believe there's very probably a way to make it work without changing any code, just with a couple of --set overrides. 

I assume you've read all of https://github.com/DefectDojo/django-DefectDojo/blob/dev/KUBERNETES.md? 

NB: for me it works using traefik ingress controller. haven't spent a lot of time trying with nginx ingress controller but I don't think it should be a problem. 

Thanks :)

File: helm/defectdojo/templates/configmap.yaml (hunk 2 above)
Comment (dsever) [reply helm/defectdojo/templates/configmap.yaml]: I'm using nginx ingress, and when  install  with this parameters, there are errors in log that comes from liveness probles, because it is actually tested by k8s engine.

File: helm/defectdojo/templates/django-ingress.yaml (hunk 1 above)
Comment (dsever) [reply helm/defectdojo/templates/django-ingress.yaml]: I had error, with this setup, I was using helm 3, not sure is that source of my issue, I will test it again

File: helm/defectdojo/values.yaml (hunk 4 above)
Comment (dsever) [reply helm/defectdojo/values.yaml]: Ignore this, I was was using same repo after creating PR

Comment (dsever) [reply helm/defectdojo/values.yaml]: Ignore this, I was using that in prod, probably ended inside of PR.

**Comment by dsever** [comment]
@ptrovatelli, I' was trying to join OWASP #slack but without success.

**Comment by dsever** [comment]
I made some mistakes and didn't proper create PR, so I will close this one. I will try with @ptrovatelli  clear some issues I'm facing with given helm.

File: helm/defectdojo/templates/django-ingress.yaml (hunk 1 above)
Comment (dsever) [reply helm/defectdojo/templates/django-ingress.yaml]: I checked documentation, alias will fly for deployment and service, but it looks like for ingress will not (at least for nginx controller)
